docs/journalist/sources.rst

```diff
@@ -3,17 +3,54 @@ Communicating with sources
 
 .. include:: ../includes/top-warning.rst
 
+SecureDrop Workstation lets journalists check SecureDrop, decrypt and securely
+view submissions, and reply to sources, all on the same computer.
+
+Once logged in, you will see a chat-interface-like screen, showing all of your
```

Comment:
chat-interface-like screen -> chat-like user interface (to avoid the slightly awkward double hyphenation)

Reply:
fixed in 1447bdd